plugins/console.js

```diff
@@ -23,7 +23,7 @@ var logForGivenLevel = function(level) {
     if (level === 'warn') level = 'warning';
     return function () {
         var args = [].slice.call(arguments);
-        Raven.captureMessage('' + args, {level: level, logger: 'console'});
+        Raven.captureMessage('' + args[0], {level: level, logger: 'console', extra: { args: args }});
```

Comment:
Re args.slice(1): I would prefer to keep all all the arguments in extras because you can have a complex object in the first argument of console.xxx() as well. You do not want to have to recommit your code just because you forgot that the first argument should be a string...